pennylane/templates/subroutines/__init__.py

```diff
@@ -16,7 +16,8 @@
 any of the characteristics of other templates (i.e. to prepare a specific state, to be repeated or to encode features).
 """
 
-from .interferometer import Interferometer
 from .arbitrary_unitary import ArbitraryUnitary
-from .single_excitation_unitary import SingleExcitationUnitary
 from .double_excitation_unitary import DoubleExcitationUnitary
+from .interferometer import Interferometer
+from .single_excitation_unitary import SingleExcitationUnitary
+from .uccsd_unitary import UCCSDUnitary
```

Comment:
Is UCCSD considered a subroutine because it is not repeated, like a layer/ansatz? I have to admit, I'm still a bit confused as to the difference between a subroutine/layer 😆 I'm glad they're all importable via `pennylane.templates`

Reply:
I think the distinction is only clear in my very personal head :P Yes, subroutines is anything that is not known as state preparation, or a repeated ansatz, or an ansatz that encodes data inputs.

Reply:
Yes, I agree with @mariaschuld, next hardware-efficient template will be more clear in that sense, it is built by adding layers.